init.zsh

```diff
@@ -9,7 +9,7 @@ if ! is-at-least 5.2; then
 fi
 
 # Define zim location
-ZIM="${ZDOTDIR:-${HOME}}/.zim"
+(( ! ${+ZIM_HOME} )) && ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
```

Comment:
This needs to be (with `export`):

    (( ! ${+ZIM_HOME} )) && export ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"

because `zmanage` invokes a new shell for the sub commands, and those would fail because there would be no value for `ZIM_HOME`.